common/lib/capa/capa/inputtypes.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, status, gettext_func=unicode):
         names = {
             'correct': _('correct'),
             'incorrect': _('incorrect'),
+            'partially-correct': _('partially correct'),
```

Comment:
a few lines below this section is the "tooltips" section, which needs one more line similar to this one to build the "This is partially correct" status tooltip.